### PR TITLE
Remove pipeline submodule

### PR DESCRIPTION
**Pull request by hanars**
We had added a submodule for the elasticsearch pipeline to the seqr repo to support local installations, but once we switched to docker-compose it was no longer necessary. Now that I have moved that repo, I updated the pipeline runner to used the moved repo, and just removed the submodule entirely rather than updating it. There were a couple of utiity fucntions in that repo that we were using for deployments, and I just copied those over

### CODE REVIEW

File: deploy/servctl_utils/yaml_settings_utils.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import collections
+import io
```
Comment (hanars) [inline deploy/servctl_utils/yaml_settings_utils.py:2]: This file is copy-pasted from the piepeline repo, so theres no need to code review it: https://github.com/broadinstitute/hail-elasticsearch-pipelines/blob/master/kubernetes/yaml_settings_utils.py

File: deploy/servctl_utils/yaml_settings_utils.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    output_file_path = os.path.join(output_base_dir, relative_file_path)
+    output_dir_path = os.path.dirname(output_file_path)
+
+    if not os.path.isdir(output_dir_path):
+        os.makedirs(output_dir_path)
+
+    try:
+        with open(output_file_path, 'w') as ostream:
+            ostream.write(yaml_string)
+    except Exception as e:
```
Comment (hanars) [inline deploy/servctl_utils/yaml_settings_utils.py:101]: Codacy found an issue: [Unused variable 'e'](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=6353594)

**Comment by hanars** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- deploy/servctl_utils/yaml_settings_utils.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=6353594&bid=20629377)

File: deploy/servctl_utils/yaml_settings_utils.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    settings["TIMESTAMP"] = time.strftime("%Y%m%d_%H%M%S")
+    settings["HOST_MACHINE_IP"] = _get_ip_address()
+
+    # process settings_file_paths
+    for settings_path in settings_file_paths:
+        yaml_string = _parse_jinja_template(settings_path, template_variables=settings)
+
+        try:
+            settings_from_this_file = yaml.load(yaml_string)
+        except yaml.parser.ParserError as e:
```
Comment (hanars) [inline deploy/servctl_utils/yaml_settings_utils.py:60]: Codacy found an issue: [Unused variable 'e'](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=6353594)

File: deploy/servctl_utils/yaml_settings_utils.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    s.connect(("8.8.8.8", 80))
+    return s.getsockname()[0]
+
+
+def _parse_jinja_template(jinja_template_path, template_variables):
+    with io.open(jinja_template_path, encoding="UTF-8") as jinja_template_file:
+        try:
+            jinja_template_contents = jinja_template_file.read()
+            yaml_string = jinja2.Template(jinja_template_contents).render(template_variables)
+        except TypeError as e:
```
Comment (hanars) [inline deploy/servctl_utils/yaml_settings_utils.py:27]: Codacy found an issue: [Unused variable 'e'](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=6353594)

**Comment by hanars** [comment]
All the codacy issues are in the copy-pasted file from the elasticsearch repo, and cleaning that file up is outside the scope of this PR

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**